frontend/src/components/report/report.jsx

```diff
@@ -49,6 +50,13 @@ class Report extends Component {
     this.props.updateReport({ "id": this.props.report._id });
   };
 
+  downdateReport = (e) => {
+    e.preventDefault();
+    console.log(this.props.report.approvals === 0);
```

Comment:
Overall, looks great! Is there a reason for this `console.log`? If this was just for testing, let's delete it.

Reply:
Removed. I forgot it that was just for testing I don't need it.